src/containerapp/setup.py

```diff
@@ -36,9 +47,10 @@
 ]
 
 # TODO: Add any additional SDK dependencies here
-DEPENDENCIES = [
-    'pycomposefile>=0.0.29'
-]
+DEPENDENCIES = ['pycomposefile>=0.0.29']
+
+# Install pack CLI to build runnable application images from source
+_ = get_pack_exec_path()
```

Comment:
Have you tested whether this installation way meets your expectations?

Reply:
@zhoxing-ms -- Yes, in local development, this executable is installed at the following location:

```
<path-to-repo>/azure-cli-extensions/src/containerapp/azext_containerapp/bin
```

When the extension is installed by users, it _should_ be installed at the following location:

```
<path-to-azure-folder>/.azure/cliextensions/containerapp/azext_containerapp/bin
```

The `path` or `path.exe` executable installed in this directory will be given appropriate permission to execute and the file's existence will be checked again before running any of the newly added functionality that requires it.